public/app/core/components/sidemenu/DropDownChild.tsx

```diff
@@ -14,12 +14,24 @@ const DropDownChild: FC<Props> = props => {
     margin-right: ${theme.spacing.sm};
   `;
 
+  const childrenMenu = child.children
```

Comment:
I would destructure children to a variable to avoid repeating a lot `child.children`.

Reply:
```suggestion
  const childrenMenu = (child.children || []).map(...)
```
Downstream you are checking for the existence of children with `{renderChildren ?`, so this should be safe.